soccer/src/soccer/ros2_temp/debug_draw_interface.cpp

```diff
@@ -0,0 +1,60 @@
+#include "debug_draw_interface.hpp"
+
+#include <spdlog/spdlog.h>
+
+#include <rj_constants/topic_names.hpp>
+
+namespace ros2_temp {
+
+DebugDrawInterface::DebugDrawInterface(Context* context, rclcpp::Executor* executor)
+    : context_(context) {
+    node_ = std::make_shared<rclcpp::Node>("_debug_draw_interface");
+    debug_draw_sub_ = node_->create_subscription<rj_drawing_msgs::msg::DebugDraw>(
+        viz::topics::kDebugDrawPub, rclcpp::QoS(10),
+        [this](rj_drawing_msgs::msg::DebugDraw::SharedPtr debug_draw) {  // NOLINT
+            spdlog::info("Got message");
+            latest_[debug_draw->layer] = debug_draw;
+        });
+
+    executor->add_node(node_);
+}
+
+void DebugDrawInterface::run() {
+    const auto& color_to_qt = [](const rj_drawing_msgs::msg::DrawColor& color) {
+        return QColor::fromRgb(color.r, color.g, color.b);
+    };
```

Comment:
why a lambda and not some free function?

Reply:
There wasn't really anywhere else this was likely to be used? I'll just make it a free function and leave it in a utils/ or something.